docs/language/pipeline-model.md

```diff
@@ -241,14 +240,14 @@ is unambiguously an aggregation and can be shortened to
 ```
 count() by id
 ```
-Likewise, a very common lean-forward use pattern is "searching" so by default,
-expressions are interpreted as keyword searches, e.g.,
+Likewise, a very common lean-forward use pattern is "searching", so with the
+use of leading `?` shorthand, expressions are interpreted as keyword searches, e.g.,
```

Comment:
The text here definitely needed to be changed (#5436). I'm less certain if we should stop mentioning `search` entirely in this section on Implied Operators. What I've done instead is still mention it but alongside explaining the `?` shorthand. From a language parsing perspective I respect that this makes it different from all the genuine "implied operators" since the `?` is still explicit. But I feel we could also justify mentioning it here in the spirit of "ways you can express your query with less typing", plus it contrasts nicely with `where`. I'm open to debate though!